lib/dynflow/persistence.rb

```diff
@@ -73,6 +73,10 @@ def find_execution_plan_counts(options)
       adapter.find_execution_plan_counts(options)
     end
 
+    def find_execution_plan_counts_after(options, timestamp)
```

Comment:
Shouldn't be the parameters switched? It seems to be inconsistent with the usages like in https://github.com/Dynflow/dynflow/pull/447/files#diff-c58910824d8fc847b39d994b727a0dd9958e732ad1551e0e9b6fa62cccc6ede8R227. I don't even know how that would not fail...

```suggestion
    def find_execution_plan_counts_after(timestamp, options)
```

Reply:
Thanks for catching that. I first had options in the first place but I had to switch it because optional arguments should be last. I forgot about this one place.